fi/features.html

```diff
@@ -0,0 +1,458 @@
+                <li><strong>Resultantit</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Differentiaalilaskenta</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/integrals/integrals.html">Integrointidokumentit</a></p>
+            <ul>
+                <li>
+                    <strong>Raja-arvot</strong>
+                    <a href="http://www.sympygamma.com/input/?i=limit%28x*log%28x%29%2C+x%2C+0%29">limit(x*log(x), x, 0) -&gt; 0</a>
+                </li>
+                <li>
+                    <strong>Derivointi:</strong>
+                </li>
+                <li>
+                    <strong>Integrointi:</strong>
+                    Se käyttää laajennettua Risch-Norman - heuristiikkaa
+                </li>
+                <li><strong>Taylor (Laurent) - sarjat</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Yhtälönratkaisu</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/solvers/solvers.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>Polynomiyhtälöt</strong></li>
+                <li><strong>Algebralliset yhtälöt</strong></li>
+                <li><strong>Differentiaaliyhtälöt</strong></li>
+                <li><strong>Differenssiyhtälöt</strong></li>
+                <li><strong>Yhtälöryhmät</strong></li>
+                <li><strong>Diofantoksen yhtälöt</strong> <a href="http://www.sympygamma.com/input/?i=diophantine%28x%2A%2A2%20-%204%2Ax%2Ay%20%2B%208%2Ay%2A%2A2%20-%203%2Ax%20%2B%207%2Ay%20-%205%29">x<sup>2</sup> &#8722; 4xy + 8y<sup>2</sup> &#8722; 3x + 7y = 5</a>, <a href="http://www.sympygamma.com/input/?i=diophantine%282%2Ax%20%2B%203%2Ay%20-%205%29">2x + 3y = 5</a></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Kombinatoriikka</h3>
+        <div>
+            <p><a href="http://docs.sympy.org/latest/modules/combinatorics/index.html">Dokumentaatio</a></p>
+            <ul>
+                <li><strong>Permutaatiot</strong></li>
+                <li><strong>Kombinaatiot</strong></li>
+                <li><strong>Ositukset</strong></li>
+                <li><strong>Aliryhmät</strong></li>
+                <li><strong>Permutaatioryhmät</strong>: Polyhedral, Rubik, Symmetric, ...</li>
+                <li><strong>Prüfer ja Gray - koodit</strong></li>
+            </ul>
+        </div>
+    </div>
+
+    <div class="dropdown">
+        <h3>Diskreetti matematiikka</h3>
+        <div>
+            <ul>
+                <li><strong>Binomikertoimet</strong></li>
+                <li><strong>Summat</strong></li>
+                <li><strong>Tulot</strong></li>
+                <li><strong>Lukuteoria:</strong> alkulukujen generointi, alkulukutestit, kokonaislukuhajotelmat, diofantoksen yhtälöt...</li>
+                <li><strong>Loogiset lauseet</strong></li>
```

Comment:
Local logicians use the term "lause" in a rather strict sense to denote logical formulas with no free variables. They would use "kaava" (formula) for the general case. My personal favourite is "lauseke" (i.e. "Loogiset lausekkeet").